src/components/MicrofactsTooltip/MicrofactsTooltip.tsx

```diff
@@ -12,113 +12,111 @@ const useStyles = makeStyles((theme: Theme) =>
       width: '500px',
       maxWidth: '90vw',
     },
+    factBoxCard: {
+      backgroundColor: '#e9edee',
+      color: '#000000',
+      boxShadow: '0px 6px 7px rgba(0, 0, 0, 0.2)',
+      backdropFilter: 'blur(7.5px)',
+      borderRadius: '13px',
+      borderBottom: '2px solid #FFE275',
+      width: '42vw',
+      maxWidth: '24rem',
+      maxHeight: '20rem',
+      overflowY: 'auto',
+      padding: '0.5em',
+    },
+    triggerBox: {
+      color: 'rgba(86, 82, 78, 1)',
+      cursor: 'help',
+      borderBottom: '1px dashed #56524E',
+      textDecoration: 'none',
+    },
   })
 );
 
-export interface MicrofactsTooltipProps {
-  entity: Entity;
-  updateEntity: (toReplace: Entity, newEntity: Entity) => void;
+interface FactboxProps {
+  entity: string;
+  description: string;
+  displayLink: string;
+  link: string;
 }
 
-export const MicrofactsTooltip = (props: MicrofactsTooltipProps) => {
+const Factbox = (props: FactboxProps) => {
   const classes = useStyles();
-  const { entity, updateEntity } = props;
+  const { entity, description, displayLink, link } = props;
+
+  return (
+    <Card className={classes.factBoxCard}>
+      <Grid container direction="column" spacing={0.5}>
+        <Grid item>
+          <Typography variant="h6">{entity}</Typography>
+        </Grid>
+        <Grid item>
+          <Typography variant="body1" fontSize={'0.9em'}>
+            {description.length > 220
+              ? description.slice(0, 220) + ' ...'
+              : description}
+          </Typography>
+        </Grid>
+        <Grid item>
+          <Typography fontFamily="DM Mono" fontSize={'0.8em'}>
+            Read more on:{' '}
+            <a href={link} target="_blank" rel="noreferrer">
+              {displayLink}
```

Comment:
Any idea why this is in focus when you open the tooltip? It adds that focus box around the link which draws more attention than needed.
<img width="96" alt="Screenshot 2022-12-01 at 12 02 56" src="https://user-images.githubusercontent.com/44401648/204982361-2ac00b3d-8b06-46be-947e-e3fb846ba04a.png">

Reply:
Fixed